README.md

```diff
@@ -92,119 +92,141 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Upgraded the System.IO.Abstractions dependency to 21.0.29. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Upgraded the WindowsAppSDK dependency to 1.6.241114003. Thanks [@shuaiyuanxx](https://github.com/shuaiyuanxx)!
- - Upgraded the MSTest dependency to 3.6.3. Thanks [@Youssef1313](https://github.com/Youssef1313)!
- - Upgraded the check-spelling CI dependency to 0.0.24 and fixed related spell checking issues. Thanks [@jsoref](https://github.com/jsoref)!
- - Removed duplicate names from the spellcheck allowed names file. Thanks [@htcfreek](https://github.com/htcfreek)!
- - Improved logging of asynchronous methods call stacks when logging an error.
- - Created a MSBuild props file to be imported by other projects to enable AOT support.
- - Made the Peek utility source code AOT compatible.
- - Updated .editorconfig rules to relax squiggly IDE errors in Visual Studio 17.12. Thanks [@snickler](https://github.com/snickler)!
- - Moved Xaml.Styler from the root to the src folder.
+ - Added fuzz testing for AdvancedPaste, with a new pipeline for OneFuzz.
+ - Added a new CI pipeline to build with the latest WindowsAppSDK.
+ - Added a new CI pipeline to build with the latest webview2 from Edge Canary.
+ - Made the HostsUILib project AOT compatible. Thanks [@snickler](https://github.com/snickler) for your help reviewing this!
+ - Made FilePreviewCommon and MarkdownPreviewHandler AOT compatible. Thanks [@snickler](https://github.com/snickler) for your help reviewing this!
+ - Made the PowerAccent.Core project AOT compatible. Thanks [@snickler](https://github.com/snickler) for your help reviewing this!
+ - Cleaned up some code for AOT compatibility in the Advanced Paste module. Thanks [@snickler](https://github.com/snickler) for your help reviewing this!
+ - Removed the prerelease flag from the PowerToys development DSC configurations. Thanks [@denelon](https://github.com/denelon)!
+ - Improved Dart CI reliability by improving error messages and retrying to the step that installs the correct dotnet version.
+ - Improved Dart CI reliability by fixing retries when downloading the localization files.
+ - Improved Dart CI build times by removing the steps to build the no longer needed abstracted utility nuget packages.
+ - Removed the solution.props file from the solution root.
+ - Fixed PowerToys Run Calculator plugin tests when running in systems with different number formats. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Updated many .NET packages from .NET 9.0.0 to 9.0.1 for security fixes. Thanks [@snickler](https://github.com/snickler)!
+ - Refactored the Mouse Without Borders Common.Log.cs and Common.Receiver.cs files. Thanks [@mikeclayton](https://github.com/mikeclayton)!
 
 #### What is being planned for version 0.88
 
-For [v0.88][github-next-release-work], we'll work on the items below:
+For [v0.89][github-next-release-work], we'll work on the items below:
 
  - Stability / bug fixes
  - New module: File Actions Menu
```

Comment:
Is this really in the planning for the next release?

Reply:
Yes, to the best of our ability :)